pkg/flink/internal/autocomplete/.snapshots/TestSetAutoCompletionSnapshot

```diff
@@ -1,10 +1,10 @@
 ([]prompt.Suggest) (len=2) {
   (prompt.Suggest) {
-    Text: (string) (len=36) "SET 'table.results-timeout' = '600';",
-    Description: (string) (len=102) "Total amount of time in seconds to wait before timing out the request waiting for results to be ready."
+    Text: (string) (len=39) "SET 'client.results-timeout' = '10000';",
+    Description: (string) (len=107) "Total amount of time in milliseconds to wait before timing out the request waiting for results to be ready."
   },
   (prompt.Suggest) {
-    Text: (string) (len=47) "SET 'table.local-time-zone;' = 'Europe/Berlin';",
-    Description: (string) (len=58) "Used to modify the configuration or list the configuration"
+    Text: (string) (len=44) "SET 'sql.local-time-zone' = 'Europe/Berlin';",
+    Description: (string) (len=48) "Used to set the timezone for the current session"
```

Comment:
maybe give a hint that this is `TZID ,GMT offset or UTC`.

Reply:
Good call, will do!